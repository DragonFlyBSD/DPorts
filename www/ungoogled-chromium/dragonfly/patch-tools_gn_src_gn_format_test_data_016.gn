diff --git tools/gn/src/gn/format_test_data/016.gn tools/gn/src/gn/format_test_data/016.gn
index 00a79922828..e0e08f864c5 100644
--- tools/gn/src/gn/format_test_data/016.gn
+++ tools/gn/src/gn/format_test_data/016.gn
@@ -1 +1 @@
-something = !is_win && is_linux || is_mac && !(is_freebsd || is_ios)
+something = !is_win && is_linux || is_mac && !(is_freebsd || is_ios || is_dragonfly)
