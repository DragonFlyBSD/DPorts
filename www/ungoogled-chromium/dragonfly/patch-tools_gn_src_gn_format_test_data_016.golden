diff --git tools/gn/src/gn/format_test_data/016.golden tools/gn/src/gn/format_test_data/016.golden
index 3f4f15bc5077..e9610a0be38c 100644
--- tools/gn/src/gn/format_test_data/016.golden
+++ tools/gn/src/gn/format_test_data/016.golden
@@ -1 +1 @@
-something = (!is_win && is_linux) || (is_mac && !(is_freebsd || is_ios))
+something = (!is_win && is_linux) || (is_mac && !(is_freebsd || is_ios || is_dragonfly))
